services/navigation/server.go

```diff
@@ -142,6 +143,24 @@ func (server *serviceServer) RemoveWaypoint(ctx context.Context, req *pb.RemoveW
 	return &pb.RemoveWaypointResponse{}, nil
 }
 
+func (server *serviceServer) GetObstacles(ctx context.Context, req *pb.GetObstaclesRequest) (
+	*pb.GetObstaclesResponse, error,
```

Comment:
[nit] no reason to line break here

Reply:
i will object to this nit. My reasoning is that I am following the convention of line breaks as displayed in this file.

Reply:
The entirely of this file does not follow our style for line breaks.

Style is for things to be on one line, except in the case where they are too long, in which case we want to break each parameter onto a separate line. We should update the file to follow the rest of RDK.